legend-engine-ide-lsp-default-extensions/src/main/java/org/finos/legend/engine/ide/lsp/extension/connection/ConnectionLSPGrammarExtension.java

```diff
@@ -92,7 +93,9 @@ protected void collectCommands(SectionState sectionState, PackageableElement ele
     @Override
     public Iterable<? extends LegendExecutionResult> execute(SectionState section, String entityPath, String commandId, Map<String, String> executableArgs, Map<String, Object> inputParams)
     {
-        return GENERATE_DB_COMMAND_ID.equals(commandId) ? generateDBFromConnection(section, entityPath) : super.execute(section, entityPath, commandId, executableArgs, Map.of());
+        return GENERATE_DB_COMMAND_ID.equals(commandId)
+               ? generateDBFromConnection(section, entityPath)
+               : FunctionExecutionSupport.execute(this, section, entityPath, commandId, executableArgs, inputParams);
```

Comment:
Dont think we should change connection extension as this does not implement the function execution support interface.

Reply:
Makes sense. Reverted the change to this file.